doc/modules/compose.rst

```diff
@@ -0,0 +1,360 @@
+
+.. _combining_estimators:
+
+==================================
+Pipelines and composite estimators
+==================================
+
+Transformers are usually combined with classifiers, regressors or other
+estimators to build a composite estimator.  The most common tool is a
+:ref:`Pipeline <pipeline>`. Pipeline is often used in combination with
+:ref:`FeatureUnion <feature_union>` which concatenates the output of
+transformers into a composite feature space.  :ref:`TransformedTargetRegressor
+<transformed_target_regressor>` deals with transforming the :term:`target`
+(:term:`y`) (e.g. log-transform) while Pipelines only transform the observed
```

Comment:
small nitpick: but you can still put a TransformedTargetRegressor inside a Pipeline .. ? (and then the pipeline transforms also y)

Reply:
That's like saying a pipeline can transform y already because it can include a classifier that centres y before regression. I don't think it appropriate to be so pedantic here.

Reply:
Well, my first reaction when reading it (being honest here) was: "oh, can a Pipeline not hold a TransformedTargetRegressor", but maybe I was thinking through too much :)

I think it is mainly the "while" that was triggering it. For example making it two separate sentences, or switching the order (first pipeline and then TTR) might already solve my confusion.

Reply:
I thought a bit like @jorisvandenbossche  but you have a good point